lib/formatters.js

```diff
@@ -1,14 +1,11 @@
 'use strict';
 
-const Os = require('os');
 const Stringify = require('fast-safe-stringify');
 
 const influxInt = (value) => {
-    if (Number.isInteger(Number(value))) {
-        return `${String(value)}i`;
-    }
-    else if (!isNaN(value)) {
-        return +value;
+    const num = parseInt(value, 10);
```

Comment:
This will only get called by things which are known to be a number. Nothing with data in the format you described should get passed to this function. See `line-protocol.js` L147-149

Reply:
That being said, I fixed it.

Reply:
👍